src/splatting/Splatting.js

```diff
@@ -1176,6 +1177,52 @@ async function main(initialFilePath) {
     }
 }
 
+let pendingCapture = null;
+
+function capture(outputWidth, outputHeight, compressionOptions) {
```

Comment:
See previous comment, also not ideal to have this level of repeated code but I guess it's okay for a little feature like this

Reply:
👆agreed. Maybe if we have time to revisit this later? Since it's a small feature.

Reply:
I'll see if there's a sensible way for me to refactor this before I merge the PR... 🤔